app/Services/Tmdb/Client/TV.php

```diff
@@ -310,11 +323,19 @@ public function __construct(int $id)
             ])
             ->json();
 
+        $this->ratings = Http::acceptJson()
+            ->withUrlParameters(['id' => $id])
+            ->get('https://api.TheMovieDB.org/3/tv/{id}/content_ratings', [
+                'api_key' => config('api-keys.tmdb'),
+            ])
+            ->json();
+
```

Comment:
This should not have been added. It should have been added to the `append_to_response` field above.